tests/metrics/network/network-nginx-ab-benchmark.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Description:
+#  Test (host<->docker) network performance using
+#  an nginx container and running the Apache 
+#  benchmarking tool in the host to calculate the 
+#  requests per second
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Ports where it will run
+port=80:80
+# Image name
+image=nginx
+# Url
+url=localhost:80
+# Number of requests to perform
+# (large number to reduce standard deviation)
+requests=100000
+# Number of multiple requests to make
+# (large number to reduce standard deviation)
+concurrency=100
+
+function setup {
+	runtime_docker
+	kill_processes_before_start
+}
+
+function check_environment {
+	check_ab=$(ab -V)
```

Comment:
You don't need to set the value of `ab -V` to a variable as `check_ab` isn't used. Just call `ab -V` and then check `$?`.